README.md

```diff
@@ -5,7 +5,14 @@ Code for recharging a robot infinitely in the 2020 FRC game Infinite Recharge
 After gradle has synced, either use the gradle menu to run
 > Tasks > help > addKtlintFormatGitPreCommitHook
 
-or in your commandline, run
+For Windows, open the commandline and run
+
+```
+chmod +x gradlew
```

Comment:
I think you should be able to push the gradlew file with the correct permissions, but might be wrong. Try chmod'ing it on a UNIX system and seeing if you can push the change.